profiles/base/package.use.mask

```diff
@@ -6,6 +6,13 @@
 
 # New entries go on top.
 
+# Ivy <openrc@posteo.de> (2025-07-11)
+# app-editors/gnome-text-editor[spell] has been dropped upstream and
+# replaced with app-text/libspelling, but will likely be added back
+# in future releases.
+app-editors/gnome-text-editor-47.2 spell
```

Comment:
This is invalid syntax. You need = if it's only for exactly that version or ~ if you want to include revisions. I guess you want ~ here.

Reply:
Huh, pkgcheck didn't complain? @arthurzam